src/convert_image/x86.rs

```diff
@@ -621,3 +633,140 @@ pub fn nv12_bt709_bgra_lrgb(
         Colorimetry::Bt709,
     )
 }
+
+pub fn rgb_lrgb_bgra_lrgb(
+    width: u32,
+    height: u32,
+    _last_src_plane: u32,
+    src_strides: &[usize],
+    src_buffers: &[&[u8]],
+    _last_dst_plane: u32,
+    dst_strides: &[usize],
+    dst_buffers: &mut [&mut [u8]],
+) -> bool {
+    if width == 0 || height == 0 {
```

Comment:
We should detect when there is not enough data in the buffers, like the other algorithms.